not related to `security', but displaying stuff as not realuid just shows a
bunch of permission errors, so display current uid only
--
--- a/lib/dialects/linux/machine.h
+++ b/lib/dialects/linux/machine.h
@@ -390,7 +390,7 @@
  * (the one that its user logged on with) of the lsof process.
  */
 
-/* #define	HASSECURITY	1 */
+#define	HASSECURITY	1
 
 /*
  * If HASSECURITY is defined, define HASNOSOCKSECURITY to allow users
